pkg/process/procutil/process_linux.go

```diff
@@ -3,19 +3,25 @@
 package procutil
 
 import (
+	"bytes"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"syscall"
+	"time"
+	"unicode"
 
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
-	"github.com/DataDog/gopsutil/host"
 )
 
 const (
+	// ClockTicks is the number of clock ticks per second
+	// C.sysconf(C._SC_CLK_TCK)
+	ClockTicks = 100
```

Comment:
Doesn't this depend on the underlying hardware?

Reply:
This is copied from gopsutil. It's definitely not perfect solution, but I read somewhere that 100 is quite common(an so far works for customers). According to https://github.com/shirou/gopsutil/issues/42 it's only possible to use cgo to collect this, but I'll look into it in the future.

Reply:
I ported the same logic from gopsutil to use `getconf` command to fetch this.